common/config.go

```diff
@@ -69,7 +68,6 @@ func MainnetConfig() *Config {
 		},
 		SwapCreatorAddr: ethcommon.HexToAddress("0xD3d19539D61bB0e7617E499C7262594E71CA1c66"),
 		// ForwarderAddr is from https://docs.opengsn.org/networks/addresses.html
-		ForwarderAddr: ethcommon.HexToAddress("0xB2b5841DBeF766d4b521221732F9B618fCf34A87"),
```

Comment:
I'm kind of unsure now about removing this hardcoded value, it's useful for needing to redeploy, and I also think we can and should skip the code check if the addresses match

Reply:
I think we should still document the contract addresses somewhere if we remove it from here

Reply:
We still have the constants, they are just in the repo with the code that creates and validates the forwarder. Moving the tiny fraction of `go-relayer` that we use into this repo, so we don't need to maintain the large amount of unused/not-so-tested code in a secondary repo, would be a code maintenance win in my opinion though. Do you want me to add back the forwarder addresses here as code comments?

Reply:
Code comment updated.